.shellcheckrc

```diff
@@ -2,5 +2,5 @@
 disable=SC1091
 # Disables the check for expanding an array, as it predictably gives first element
 disable=SC2128
-# Disbale check for var not set in same file
+# Disabales check for var not set in same file
```

Comment:
Almost correct ;-) ."Disbale" => "Disabales"...should be "Disables".

Reply:
😆 my brain clearly wasn't working